modules/user/client-react/components/MetaData.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import Helmet from 'react-helmet';
+
+import settings from '../../../../settings';
+
+interface MetaDataProps {
+  title: string;
+  meta: string;
+}
+
+const MetaData = ({ title, meta }: MetaDataProps) => (
```

Comment:
Why `MetaData` is located in the `user`? Is it the only module that will use it?